src/net.cpp

```diff
@@ -1439,6 +1446,36 @@ bool CConnman::GenerateSelectSet(std::set<SOCKET> &recv_set, std::set<SOCKET> &s
     return !recv_set.empty() || !send_set.empty() || !error_set.empty();
 }
 
+#ifdef USE_EPOLL
+void CConnman::SocketEventsEpoll(std::set<SOCKET> &recv_set, std::set<SOCKET> &send_set, std::set<SOCKET> &error_set, bool fOnlyPoll)
+{
+    const size_t maxEvents = 64;
+    epoll_event events[maxEvents];
+
+    wakeupSelectNeeded = true;
+    int n = epoll_wait(epollfd, events, maxEvents, fOnlyPoll ? 0 : SELECT_TIMEOUT_MILLISECONDS);
+    wakeupSelectNeeded = false;
+    if (n > 16) {
+        LogPrintf("CConnman::%s -- epoll_wait returned %d\n", __func__, n);
+    }
+    for (int i = 0; i < n; i++) {
```

Comment:
```suggestion
    for (int i = 0; i < n; ++i) {
```

Reply:
Hehe, is this still recommended in literature? It doesn't make a real difference anymore as compilers got too intelligent for such manual optimizations.

Reply:
Yeah, dev notes does say that pre-inc is prefered :joy:

Reply:
We should maybe remove this then...or change it to apply only to iterators.